src/components/Answers/AnswerItem.js

```diff
@@ -73,4 +76,4 @@ const AnswerItem = ({
     </>
   );
 };
-export default AnswerItem;
+export default React.memo(AnswerItem);
```

Comment:
it's not necessary to use React.memo here, is there a specific reason this is in React.memo?